README.md

```diff
@@ -103,9 +103,21 @@ This means that something is wrong with the ROM's contents. Either the baserom f
 
 Running `make init` will also make the `./expected` directory and copy all of the files there, which will be useful when running the diff script. The diff script is useful in decompiling functions and can be ran with this command: `./tools/asm-differ/diff.py -wmo3 <insert_function_here>`
 
```

Comment:
I changed the format slightly to be consistent with the next note (`**Note**:`), and added an example `make -j4` (I think it can be confusing to convey that the `N` is not literal)